ROADMAP.md

```diff
@@ -0,0 +1,92 @@
+# io.js Roadmap
+
+***This is a living document, it describes the policy and priorities as they exist today but can evolve over time.***
+
+## Stability Policy
+
+The single more important consideration in every code change is the impact it will have, positive or negative, on the ecosystem (modules and applications).
+
+io.js does not remove stdlib JS API.
+
+Shipping with current and well supported dependencies is the best way to ensure long term stability of the platform. When those dependencies are no longer maintained io.js will take on their continued maintenance as part of our [Long Term Support policy](#long-term-support).
+
+io.js will continue to adopt new V8 releases.
+* When V8 ships a breaking change to their C++ API that can be handled by [`nan`](https://github.com/rvagg/nan)
+the *minor* version of io.js will be increased.
+* When V8 ships a breaking change to their C++ API that can NOT be handled by [`nan`](https://github.com/rvagg/nan)
+the *major* version of io.js will be increased.
+* When new features in the JavaScript language are introduced by V8 the
+*minor* version number will be increased. TC39 has stated clearly that no
+backwards incompatible changes will be made to the language so it is
+appropriate to increase the minor rather than major.
+
+No new API will be added in *patch* releases.
+
+Any API addition will cause an increase in the *minor* version.
+
+### Long Term Support
+
+`io.js` supports old versions for as long as community members are fixing bugs in them.
+
+As long as there is a community back porting bug fixes we will push patch releases for those versions of `io.js`.
+
+When old versions of dependencies like V8 are no longer supported by their project `io.js` will take on the responsibility of maintenance to ensure continued long term support in `io.js` patch releases.
```

Comment:
Even in the latest version this isn't very clear. There's a big difference between the first two paragraphs which are "as long as the community submits patches we will do releases" and the last one which is "io.js will take on the responsibility of maintenance for all old V8 releases that we bundle in our releases." It should be more clear that the latter is just a facet of the former. E.g. "the same process applies for dependencies. If, for example, the community wants to help backport security patches from later V8 versions to those shipped with old io.js releases, we will readily take them and float them onto the version of V8 found in the appropriate io.js branch."